### PR TITLE
[docs] Added Missing Hints in policy-reference

### PR DESCRIPTION
**Pull request by Pushkarm029**
### Why the changes in this PR are needed?
- To improve user experience of the documentation.

### What are the changes in this PR?
- added # with `import future.keywords.in` where **in** is used.
- added # with `import future.keywords.if` where **if** is used.
- added # with `import future.keywords.contains` where **conatins** is used.

### CODE REVIEW

File: docs/content/policy-reference.md [hunk 1]
```diff
@@ -243,7 +246,7 @@ a := 100 if { ... }
 a_set[x] { ... }
 a_set[y] { ... }
 
-# alternatively, with future.keywords.contains and future.keywords.if
+# alternatively, with `future.keywords.contains` and `future.keywords.if`
```
Comment (anderseknert) [inline docs/content/policy-reference.md:249]: Perhaps we should add "import" here for consistency?

File: docs/content/policy-reference.md [hunk 2]
```diff
@@ -286,6 +291,7 @@ f(x) := "C" if { x >= 70; x < 80 }
 ### Reference Heads
 
 ```live:rules/ref_heads:module:read_only
+# with `future.keywords.contains` and `future.keywords.if`
```
Comment (anderseknert) [inline docs/content/policy-reference.md:294]: Same here.

**Comment by anderseknert** [comment]
Thanks!

**Comment by Pushkarm029** [comment]
@anderseknert can you recommend some code related issues for me?

**Comment by anderseknert** [comment]
This looks like a potential candidate https://github.com/open-policy-agent/opa/issues/5848

**Pull request merged**